src/3box.js

```diff
@@ -529,16 +522,68 @@ class Box extends BoxApi {
   }
 }
 
-function initIPFSRepo () {
+function createIframeCache () {
```

Comment:
This function should probably be called on pageload. Otherwise loading the iframe will slow things down.

Reply:
cc @zachferland for input.

Reply:
yeah, if in browser env, should optimistically load on pageload, and then if any configuration options passed later on function call differ, reload or ignore (mostly want to optimize default then can add different configuration opts/paths if people need it) 

think we could look to move most of this code out of 3boxjs as well, in attempt to make changes as minimal as possible here and for all iframe code to be more contained. I recently did took that path for idw iframe, after doing it more like this last time, before release of both we can try to share some of the same patterns. Lib could export iframe cache service, with similar interface as well (create, func to get orbit proxy, and func to get ipfs proxy). have to see if anything makes that difficult though